### ISSUE TITLE
Future work

### ISSUE THREAD

**Issue by jmatak** [josipjukic/Adversarial-NLP#1]
I see "In progress..." tag on your `README.md`. When will your code be fully finished and publically available to use?  Do you plan to expose your research as a functional framework in the future?  Looking forward to your work!

**Comment by josipjukic** [comment]
Thank you for your comment. The project is nearly finished and a demonstration site (Django framework) is in the making. The plan is to push all of the instructions for _local build_ during July, 2020. It will be kept fairly simple, with few examples and a possibility to play with various attacks on any kind of NLP models that meet the minimal requirements (e.g. standardized names for special methods such as `predict` or `predict_proba`). In addition, one can test the robustness of a certain model by evaluating how well it copes with the attacks (this functionality will be integrated into the site). As a counterpart, a similar set of functionalities will be provided in the form of Python scripts. Furthermore, an another Jupyter notebook will be added with examples and instructions on how to conduct the experiments by yourself or just try out some of the attacks.
In summary, it is not intended to make a full-blown framework out of this project, but there will certainly be plenty of examples to see and try out with the attacks on your custom data. Hopefully, the site will provide some interesting results and insights, with the additional possibility of running experiments if one is interested more profoundly.
